doc/tutorials/lris_howto.rst

```diff
@@ -0,0 +1,485 @@
+
+See :ref:`pypeit-chk-wavecalib` for a detailed description of all the columns.
+Note that the slits with ``SpatID`` 384 and 474 have all the values set to 0.0. This is because
+these slits are actually alignment boxes and they are not reduced by the pipeline.
+
+Tilts
+:::::
+
+Wavelength tilts are measured performing a 2D fit to the traced arc lines.
+There are several QA files written for the 2D fits. One example is
+``QA/PNGs/Arc_tilts_2d_A_0_DET01_S0212.png``:
+
+.. image:: ../figures/lris_arc2d.png
+  :scale: 20%
+
+Each horizontal line of black open circles identifies a traced arc line.
+Red circles shows the points that were rejected in the 2D fitting, and the
+black points show the model predicted position.  The text across the top of
+the figure gives the RMS of the 2D wavelength solution, which should be less than 0.1 pixels.
+
+The 2D fit for the wavelength tilts can also be inspected using the script :ref:`pypeit_chk_tilts`,
+which shows a :ref:`tiltimg` image in a `ginga`_ or `matplotlib`_ window with the
+traced and 2D fitted tilts over-plotted. Here is an example:
+
+.. code-block:: bash
+
+    pypeit_chk_tilts Calibrations/Tilts_A_0_DET01.fits
+
+.. image:: ../figures/lris_tilts.png
+   :scale: 40%
+
+This shows a zoom-in of a :ref:`tiltimg` image in a `ginga`_ window with overlaid
+the 2D fitted tilts (blue), the masked pixels (red), and the pixels rejected in the 2D fitting (yellow).
+
+See :ref:`tilts` for further details.
+
+Flatfield
++++++++++
+
+PypeIt computes a number of multiplicative corrections to correct the 2D spectral response
+for pixel-to-pixel detector throughput variations and lower-order spatial and spectral illumination
+and throughput corrections.  We collectively refer to these as flat-field corrections; see
+:ref:`here <flat_fielding>` and :ref:`here <flat>`.
+To inspect the ``Flat`` images we can use the script :ref:`pypeit_chk_flats`, with this explicit call:
+
+.. code-block:: bash
+
+    pypeit_chk_flats Calibrations/Flat_A_0_DET01.fits
+
+Here is a zoom-in screenshot from the first tab in the `ginga`_ window (``pixflat_norm``):
+
+.. image:: ../figures/lris_flat.png
+   :scale: 40%
+
+This shows the normalized flat field image. The green/magenta lines are the slit edges, which are
+tweaked using the illumination flat field.
+
+
+See :ref:`flat` and :ref:`flat_fielding` for further details.
+
+Object finding
+--------------
+
+After the above calibrations are complete, PypeIt will iteratively identify
+sources, perform global and local sky subtraction, and perform 1D spectral
+extractions.  This process is fully described here: :ref:`object_finding`.
+
+PypeIt produces QA files that allow you to assess the detection of the objects.
+One example is ``QA\PNGs\r230417_01033-frb22022_LRISr_20230417T082242.672_DET01_S0212_obj_prof.png``:
+
+.. image:: ../figures/lris_objfind.png
+   :scale: 30%
+
+This shows the spatial profile of the object's S/N collapsed along the spectral direction.
```

Comment:
Can we comment briefly only what a bad case might look like?  Or at least what we hope to see?

Reply:
done